cheatsheets_excluded/Content_Security_Policy_Cheat_Sheet.md

```diff
@@ -1,203 +1,199 @@
+- `default-src` is a fallback directive for the other fetch directives. Directives that are specified have no inheritance, yet directives that are not specified will fall back to the value of `default-src`.
+- `child-src` allows the developer to control nested browsing contexts and worker execution contexts.
+  - `frame-src` specifies the URLs which can be loaded into nested browsing contexts (*e.g.* `<iframe>`).
+  - `worker-src` specifies the URLs which can be loaded as worker, sharedworker, or serviceworker. Fallback's on `script-src` too.
+- `connect-src` provides control over fetch requests, XHR, eventsource, beacon and websockets connections.
+- `font-src` specifies which URLs to load fonts from.
 
--   **connect-src** (d) - restricts which URLs the protected resource can load using script interfaces. (e.g. send() method of an XMLHttpRequest object)
--   **font-src** (d) - restricts from where the protected resource can load fonts
--   **img-src** (d) - restricts from where the protected resource can load images
--   **media-src** (d) - restricts from where the protected resource can load video, audio, and associated text tracks
--   **object-src** (d) - restricts from where the protected resource can load plugins
--   **script-src** (d) - restricts which scripts the protected resource can execute. Additional restrictions against, inline scripts, and eval. Additional directives in CSP2 for hash and nonce support
--   **style-src** (d) - restricts which styles the user may applies to the protected resource. Additional restrictions against inline and eval.
--   **default-src** - Covers any directive with *(d)*
--   **frame-src** - restricts from where the protected resource can embed frames. Note, deprecated in CSP2
--   **report-uri** - specifies a URL to which the user agent sends reports about policy violation
--   **sandbox** - specifies an HTML sandbox policy that the user agent applies to the protected resource. Optional in 1.0
+- `img-src` specifies the URLs that images can be loaded from.
+- `manifest-src` specifies the URLs that application manifests may be loaded from.
+- `media-src` specifies the URLs from which video, audio and text track resources can be loaded from.
+- `prefetch-src` specifies the URLs from which resources can be prefetched from.
+- `object-src` specifies the URLs from which plugins can be loaded from.
+- `script-src` specifies the locations from which a script can be executed from. It is a fallback directive for other script-like directives.
+  - `script-src-elem` controls the location from which execution of script requests and blocks can occur.
+  - `script-src-attr` controls the execution of event handlers.
+- `style-src` controls from where styles get applied to a document. This includes `<link>` elements, `@import` rules, and requests originating from a `Link` HTTP response header field.
+  - `style-src-elem` controls styles except for inline attributes.
+  - `style-src-attr` controls styles attributes.
```

Comment:
It is not listed [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src) add your source and clarify the directive (I don't know it 😃 )

Reply:
OK, my mistake, sorry.

Btw, I have found [this post](https://groups.google.com/a/chromium.org/d/msg/blink-dev/uauMQQQ7fAI/UAY5FX9UBgAJ) that explains usage of `script-src-attr`, `script-src-elem`, `style-src-attr`, `style-src-elem` directives.

Perhaps can we add this link as help source, what do you think?